v-next/hardhat-viem/src/accounts.ts

```diff
@@ -0,0 +1,22 @@
+import type { EthereumProvider } from "@ignored/hardhat-vnext/types/providers";
```

Comment:
This file and the others, excluding `index.ts` and the type files should move to an `./internal` folder.

Reply:
Done [here](https://github.com/NomicFoundation/hardhat/pull/5825/commits/04b4261526f9da3ba62cd946f6c063bbffbf0681). We should revisit this after the alpha, as the structure is not consistent across plugins.